alertnow/manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "cdb258cc-5e74-4fa2-be21-1489375bb370",
+  "app_id": "alertnow",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows"
+  ],
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "",
+    "title": "AlertNow",
+    "media": []
+  },
+  "author": {
+    "name": "AlertNow",
+    "homepage": "https://service.opsnow.com",
+    "sales_email": "sales@opsnow.com",
+    "support_email": "support@opsnow.com"
+  },
+  "oauth": {},
+  "assets": {
+    "integration": {
+      "source_type_name": "AlertNow",
+      "configuration": {},
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "alertnow.",
+        "check": "",
```

Comment:
Does this integration emit metrics? If so, can you add a metric to check (which will indicate whether the integration is installed or not)? If the integration does not send metrics, you can remove this section

Reply:
This integration does not collect or export metrics.